Update ingest-json-formats.md (#1716)
MicrosoftDocs/dataexplorer-docs: data-explorer/ingest-json-formats.md

* `multijson`: Multi-lined JSON. The parser ignores the line separators and reads a record from the previous position to the end of a valid JSON.

> [!NOTE]

> > When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ingest data in JSON format, thus ignoring ill-formatted rows errors (only the bad line will be ignored, the rest of the file will be ingested fully). If you leave this check box unselected, the data will be ingested in multijson format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).

Reviewer: ```suggestion
> When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ingest data in JSON format. This selection ignores ill-formatted rows errors by ignoring the incorrect line, and fully ingesting the rest of the file. If you leave this check box unselected, the data will be ingested in multijson format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).
```

> > When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ingest data in JSON format. This selection ignores ill-formatted rows errors by ignoring the incorrect line, and fully ingesting the rest of the file. If you leave this check box unselected, the data will be ingested in multijson format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).

Reviewer 2: ```suggestion
> When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ignore rows that are incorrectly formatted, but fully ingest the rest of the data in JSON format. Otherwise, the data is ingested in MultiJSON format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).
```

Reviewer 3: I would probably phrase this as:

When ingesting using the [ingestion wizard](ingest-data-wizard.md), the default format flavor is `multijson`.
This format allows multiline JSON records, as well as arrays of JSON records. When parsing error is encountered, the entire file is discarded.
If you are using JSON-line format (JSON records are single line, and every line is a well-formed JSON record), and would like to accommodate for an occasional malformed record, select **Ignore data format errors** to skip the malformed records while ingesting the valid ones.

* `multijson`: Multi-lined JSON. The parser ignores the line separators and reads a record from the previous position to the end of a valid JSON.

> [!NOTE]
> When ingesting using the [ingestion wizard](ingest-data-wizard.md), the default format is `multijson`. The format can handle multiline JSON records and arrays of JSON records. When a parsing error is encountered, the entire file is discarded.
> 

> > If you are using the JSON-line format, where each line is a single well-formatted JSON record, and you want to be able to handle records that are not well-formed, you can select the option to "Ignore data format errors." This will allow the valid records to be ingested while skipping the ones that are not well-formed.

Reviewer 2: ```suggestion
> If you're using the JSON Line format, where each line is a single well-formatted JSON record, and you want to be able to handle records that are not well-formed, you can select the option to "Ignore data format errors." This will allow the valid records to be ingested while skipping the ones that are not well-formed.
```
